esmvalcore/cmor/table.py

```diff
@@ -297,7 +309,18 @@ def __init__(self, table_type, short_name):
 
         self._json_data = None
 
-    def read_json(self, json_data):
+    def copy(self):
+        """
+        Returns a shalow copy of VariableInfo
+        Returns
+        -------
+        VariableInfo
+           Shallow copy of this object
+        """
+        from copy import copy
+        return copy(self)
```

Comment:
do you really need to create a function that returns a copy, can just call `copy` on the fly

Reply:
Just for convenience, in case we need something special later (or to avoid writing extra imports, I am that lazy)

Reply:
+1 for the reply :grin:

Reply:
Any particular reason not to put the import at the top of the module?

Reply:
Chanegd